cves/CVE-2014-7809.yml

```diff
@@ -174,8 +191,10 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: | 
+        This vulnerability lies in their protection from Cross Site Request Forgery
+        and this is done out of distrust of inputs.
```

Comment:
I would argue the nature of this vulnerability is not `distrust inputs` but rather `frameworks are optional`. In this case, the "framework" is simply the Java language itself. That being said, the default random number generator class was used in place of a more secure option. This implies either ignorance or misunderstanding on behalf of the developers with regards to how secure Java's number generator truly is.

Reply:
I think I initially thought of it from the perspective of not trusting the authentication tokens, but this suggestion makes a lot more sense.